src/components/Modal/ModalSettings.js

```diff
@@ -0,0 +1,88 @@
+import styled from 'styled-components';
+import { useDispatch } from 'react-redux';
+
+import { setColor, setFont } from './../../redux/modalSlice';
+import { theme } from './../styles/theme';
+import { Flex } from './../styles/styledElements';
+
+function ModalSettings({ data }) {
+    const { title, buttons } = data;
+    const dispatch = useDispatch();
+
+    const dispatchData = (font, color) => {
+        if (font) dispatch(setFont(font));
+        if (color) dispatch(setColor(color));
```

Comment:
codequality tools like `sonarqube` dislike statements without curly-brackets.

it's easier to introduce hard-to-see bugs